TeaDiary/Scripts/App/Search/Index.js

```diff
@@ -9,7 +9,7 @@ function onSearchClick() {
     search($("#SearchRequest").val(),
         $("#IsStrictSearch").prop("checked"),
         (teaList) => {
-            var displayer = new TeaListDisplayer("#SearchResults");
+            var displayer = new teaListDisplayer("#SearchResults");
```

Comment:
Why? Didn't we agree to name classes in PascalCase?

Reply:
I forgot about it >_< There are too many naming conventions